.github/workflows/test.yml

```diff
@@ -81,3 +81,7 @@ jobs:
     - name: Run Ibis expression tests
       if: ${{ matrix.producer == 'ibis' }}
       run: pytest substrait_consumer/functional/queries/ibis_expressions/
+
+    - name: Run Ibis expression tests
```

Comment:
This is the same name as above -- should it be different?

Reply:
Yes, good catch.